### DISCUSSION TITLE
Is there a way to dynamically pass the token on REST authentication

### DISCUSSION THREAD

**Discussion by genogeno** [Budibase/budibase#10731] [Help] [answered]
Hi,

Is there a way to dynamically pass value to the Bearer Token aside from static value and {{ Current User.OAuthToken }}

<img width="545" alt="image" src="https://github.com/Budibase/budibase/assets/358383/8ab79714-3754-4559-95ae-e19f0ea47b94">

Thanks!

**Comment by melohagan** [comment, marked as answer]
Hey @genogeno 

I am curious as to what your use-case is that you require the ability to dynamically pass the Bearer token?

As the Bearer token is simply a header, you could add the header within the REST query and provide a binding that way: https://docs.budibase.com/docs/rest-bindings#headers-example

There is also [dynamic variables](https://docs.budibase.com/docs/rest-variables#dynamic-variables) which allows you to chain queries, if there are refresh tokens involved?
